packages/base-contract/src/subscription_manager.ts

```diff
@@ -171,8 +171,8 @@ export class SubscriptionManager<ContractEventArgs, ContractEvents extends strin
         if (this._blockAndLogStreamerIfExists === undefined) {
             throw new Error(SubscriptionErrors.SubscriptionNotFound);
         }
-        this._blockAndLogStreamerIfExists.unsubscribeFromOnLogAdded(this._onLogAddedSubscriptionToken as string);
-        this._blockAndLogStreamerIfExists.unsubscribeFromOnLogRemoved(this._onLogRemovedSubscriptionToken as string);
+        this._blockAndLogStreamerIfExists.unsubscribeFromOnLogsAdded(this._onLogAddedSubscriptionToken as string);
```

Comment:
Looks like 7.0.0 now calls back with an [Array of logs](https://github.com/ethereumjs/ethereumjs-blockstream/commit/7ac65170c054db83eac51a731388816879911a41) rather than calling back with a single log. This type casting should result in a test failure.

Reply:
nice catch @dekz . Looks like `subscribeToOnLogsAdded/Removed` still returns a string, so the subscriptionTokens are still strings. But the callbacks take an array of logs instead of a single log. I think I fixed it in ec2ca78a4a25ebc885bb5c6d9d1a08999bd25ace

Thanks for calling my attention to this, wouldn't have noticed it otherwise :/